lms/djangoapps/courseware/access.py

```diff
@@ -160,11 +181,10 @@ def _can_access_descriptor_with_start_date(user, descriptor, course_key):  # pyl
             descriptor,
             course_key=course_key
         )
-        return (
-            descriptor.start is None
-            or now > effective_start
-            or in_preview_mode()
-        )
+        if descriptor.start is None or now > effective_start or in_preview_mode():
+            return ACCESS_GRANTED
+
+        return StartDateError("{:%B %d, %Y}".format(descriptor.start))
```

Comment:
Also, is there a possibility that the `format` will raise an exception if the start date value is not as expected?  @mekkz you had worked with this field most recently.  Any thoughts?